src/main/data/BigBed.js

```diff
@@ -53,13 +53,21 @@ function reverseContigMap(contigMap: {[key:string]: number}): Array<string> {
 }
 
 
-function extractFeaturesFromBlock(buffer, dataRange, block): ChrIdBedRow[] {
+function extractFeaturesFromBlock(buffer, dataRange, block, isCompressed): ChrIdBedRow[] {
```

Comment:
would you mind annotating these parameters with flow types while at it? If they are already obvious, feel free to ignore this comment ;)